Source/abci/Importer/aiContext.cpp

```diff
@@ -329,6 +330,7 @@ bool aiContext::load(const char *in_path)
             Alembic::AbcCoreOgawa::ReadArchive archive_reader(m_streams);
```

Comment:
Do you know you can use the IFactory class to do the whole hdf5/ogawa dance for you?
```
  IFactory	factory;
  IFactory::CoreType	archive_type;
  m_archive = factory.getArchive(m_path.c_str(), archive_type);
  m_isHDF5 = (archive_type == IFactory::kHDF5);
```